scripts/python/packages/parthenon_tools/parthenon_tools/phdf_diff.py

```diff
@@ -92,6 +93,12 @@ def processArgs():
         action="store_true",
         help="Ignore differences in metadata.",
     )
+    parser.add_argument(
+        "-f",
```

Comment:
`-f` feels weird here. Maybe just skip the short argument for this one?

Reply:
Yeah, I agree. I thought I had to provide a short argument.